### DISCUSSION TITLE
Build 2301 & 2302 breaks WSA on my system.

### DISCUSSION THREAD

**Discussion by dPoz** [microsoft/WSA#267] [General]
Hello,

Can someone provide some help on where to go regarding this issue?

Build 2211 works fine on my system. As soon as it updates to build 2301 (automatically), it has the promp/spash screen that andriod is starting until it times out (a few minutes) and closes. I cant get any other android visual out of WSA after that. Something seems to be running, as after that point, I can "turn off" WSA and it spins for a bit. Process explorer shows the process going with a bit of cpu usage. The main WSA settings pannel opens up, but no android apps. 

I tried the repair and reset (which caused me to lose my current wsa instance before I discovered how to back it up). 
I tried uninstalling it and reinstalling (dozens of times via various methods). 
The auto update does not take it to 2302, but I downloaded the appx and installed it. Still no luck. 
I have found that installing the 2211 appx version fixes the issue. The version that worked before I updated and broke wsa. The problem is that it auto updates to 2301 very soon after and then stops working again.

From what I can tell after getting 2211 to work is that there are two promps that 2301 does not do: new firewall app request or ask permission to trust my device (when trying to install something with apk installed from the windows store). 2302 did ask for a new firewall policy, its possible 2301 also asked a few weeks ago when it updated and wsa broke and I forgot that it asked.

Is there a proper support fourm for this? 
What info can I gather to help someone smart troubleshoot this?

Was there a spec change between 2211 and 2301? Some new security enforcement that my device does not satisfy for 2301?

Thank you!

**Comment by jaholme** [comment]
The best way to help diagnose issues related to WSA not starting is to file a Feedback Hub report(Windows Key + F) under the Apps > Windows Subsystem for Android. Under section 4. Add more details, be sure to include a recording of launching WSA as that will collect logs
